README.md

```diff
@@ -442,6 +444,12 @@ The `firebaseAdminInitConfig.credential.privateKey` cannot be defined on the cli
 >
 > See [this Vercel issue](https://github.com/vercel/vercel/issues/749#issuecomment-707515089) for more information.
 
+**firebaseAdminDefaultCredential**
```

Comment:
I think "useFirebaseAdminDefaultCredential" is a clearer name for a boolean.

Reply:
Agreed, changed